dev-util/xdelta/xdelta-3.1.0.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A binary diff tool that uses the VCDIFF (RFC 3284) format and compression"
+DESCRIPTION="Xdelta is a tool and library for differential compression. Xdelta \
+release version 3 supports VCDIFF encoding and decoding."
+HOMEPAGE="http://xdelta.org/"
+COPYRIGHT="2001-2015 Josh MacDonald"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/jmacd/xdelta/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="7515cf5378fca287a57f4e2fee1094aabc79569cfe60d91e06021a8fd7bae29d"
+SOURCE_DIR="xdelta-$portVersion/xdelta3"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	xdelta$secondaryArchSuffix = $portVersion
+	cmd:xdelta3$secondaryArchSuffix = $portVersion
+	lib:xdelta3$secondaryArchSuffix = $portVersion
```

Comment:
**Oops,** sorry, I juist realized that the devel package was empty because this was missing in INSTALL:
```
	packageEntries devel $developDir
```
We need to test with it, before we remove the _devel package.